### ISSUE TITLE
Cosmos DB query string syntax highlighting uses Light theme colors when Dark theme is selected, unless Dark theme is selected when the Query tab is already open

### ISSUE THREAD

**Issue by scholtes** [microsoft/AzureStorageExplorer#676]
**Storage Explorer Version**: 1.4.1
**Build Number**: 20180822.1
**Platform/OS Version**: Windows 10 Enterprise (10.0.14393 Build 14393)
**Architecture**: ia32
**Regression From**: not determined

#### Steps to Reproduce: ####
1. Open Microsoft Azure Storage Explorer
2. Select menu "View > Themes > Dark"
3. Navigate to a Cosmos DB account in the Explorer pane on the left
4. Fold open a Database within the previously opened Cosmos DB account using the Explorer tree view
5. Fold open a Collection within the previously opened Database using the Explorer tree view
6. Right click the "Documents" object under the previously selected Collection
7. Select "Open Query Tab" from the dropdown menu
8. **Observe that the query text shows up using the "Light (default)" theme colors (white background)**
9. Select menu "View > Themes > Light (default)"
10. Select menu "View > Themes > Dark"
11. **Observe that the query text shows up using the "Dark" theme colors (dark grey background)**
12. Close the opened Query tab
13. Repeat steps 6 and 7 above (re-open the Query tab) 
14. **Observe that the query text shows up using the "Light (default)" theme colors (white background)**

#### Expected Experience: ####
At steps 8, 11, and 14 above, the Query text should use Dark theme colors (dark grey background).  

#### Actual Experience: ####
In step 11 above, the Query text correctly uses Dark theme colors.  At steps 8 and 14, the Query text incorrectly uses Light (default) theme colors (white background).  

**TL;DR** the query text in a Query Tab for a Cosmos DB collection only shows up using the correct Dark theme colors when Dark theme was selected from the View>Themes menu while the query tab is already open.  If a query tab is opened *after* Dark theme is already enabled, it uses the wrong colors.

**Comment by scholtes** [comment]
Similar behavior can be seen while viewing the contents of a document.  Rather than report a separate issue I will include the behavior here.  

1. Perform steps 1 through 5 above (open a collection) 
2. Click on "Documents"
3. Click on an individual document in the list that shows up

Observe that if Dark mode is already selected, the syntax highlighting for the JSON within the document shows up as white, but switching the theme to Light (default) and back to Dark while the document is open causes the document to appear with the proper (Dark) color highlighting.

**Comment by JPhillipBrooks** [comment]
Hello. Would it be possible to please get this prioritized? I realize it's just cosmetic but it's a nuisance to apply the workaround of switching back to light theme then again back to dark theme. Fixing this would really make the product feel more polished, in my opinion.

As of v1.13.1, opening a new Cosmos DB Query tab or Documents tab still does not honor the dark theme. See below (our product's name redacted)

![image](https://user-images.githubusercontent.com/43281913/82597746-0e312b80-9b6f-11ea-99d2-df2301cc01b3.png)

**Comment by aliuy** [comment]
Azure Storage Explorer support for Azure Cosmos DB has been discontinued as of v1.24.0. We recommend users to use Azure Cosmos DB’s Data Explorer, which can be found in [Azure Portal](https://portal.azure.com/) or as a stand-alone version in http://cosmos.azure.com/ for the best Azure Cosmos DB experience. You can also find additional Azure Cosmos DB tooling here: [Azure Cosmos DB Tools](https://developer.azurecosmosdb.com/tools)
